anima-jws-voucher.mkd

```diff
@@ -278,14 +260,13 @@ Thanks for the supporting PoC implementations to Hong Rui Li and He Peng Jia.
 
 # Examples
 
-These examples are folded according to {{RFC8792}} Single Backslash rule.
+These examples are folded according to the {{RFC8792}} Single Backslash rule.
 
-## Example Pledge-Voucher-Request - PVR (from Pledge to Registrar)
-The following is an example request sent from a Pledge to the Registrar, in "General JWS JSON Serialization".
+## Example Pledge-Voucher-Request (PVR)
```

Comment:
This document is agnostic to the BRSKI-PRM exchanges. Hence, I removed the who sends to whom from the headings, but indicated in the text.

Reply:
It is fine with the change but I also did not see a problem with the original text, as the PVR is created by the pledge and provided to the registrar or the registrar agent.